railties/lib/rails/api/task.rb

```diff
@@ -10,13 +10,15 @@ class Task < RDoc::Task
         "activesupport" => {
           include: %w(
             README.rdoc
+            lib/active_support.rb
```

Comment:
By including these there is a side-effect that the methods defined in those files are now considered public API if they are not otherwise `:nodoc:`'d. That may have been intentional, however. Just pointing it out.

Reply:
I'll `:nodoc:` those.

Reply:
@zzak If any of those methods are considered private we should `:nodoc:` them.
Other generators like Rdoc Info do include these files and will consider it public API:
https://www.rubydoc.info/github/rails/rails/ActiveSupport.cache_format_version=

For some methods we already add `:nodoc:`:
https://github.com/rails/rails/blob/80c51c1d2f268cef88728abbca82c15d1c25792c/activesupport/lib/active_support.rb#L97-L101

So I guess this fixes a bug where public API methods weren't shown before.

Reply:
My really honest guess is that API being visible on the website is secondary to the `:nodoc:` it might have in the source code in terms of weight it has over whether or not it's intended to be public. This is a fair assumption IMO.

If there is no `:nodoc:` it should be considered public otherwise. After that it's either find the person with context or can make a decision about the visibility or take a wild guess :joy: